sites/partners/src/listings/PaperListingForm/sections/Units.tsx

```diff
@@ -175,6 +180,23 @@ const FormUnits = ({ listing, unitsSummaries, setSummaries, disableUnitsAccordio
         grid={false}
         separator
       >
+        <GridSection columns={3} className={"pb-10"}>
+          <GridCell>
+            <p className="field-label">{t("listings.homeType")}</p>
```

Comment:
You should be able to wrap this in `<ViewItem>` so the styling is consistent elsewhere, where we usually wraps selects in that component